lhotse/cut.py

```diff
@@ -113,6 +113,38 @@ def plot_features(self, root_dir: Optional[Pathlike] = None):
         features = np.flip(self.load_features(root_dir=root_dir).transpose(1, 0), 0)
         return plt.matshow(features)
 
+    def supervisions_feature_mask(self) -> np.ndarray:
+        """
+        Return a 1D numpy array with value 1 for frames covered by at least one supervision,
+        and 0 for frames not covered by any supervision.
+        """
+        assert self.has_features, f"No features available. " \
+                                  f"Can't compute supervisions feature mask for cut with ID: {self.id}."
+        mask = np.zeros(self.num_frames)
+        for supervision in self.supervisions:
+            st = round(supervision.start / self.frame_shift) if supervision.start > 0 else 0
+            et = round(supervision.end / self.frame_shift) if supervision.end < self.duration else self.num_frames
+            mask[st:et] = 1
+        return mask
+
+    def supervisions_audio_mask(self) -> np.ndarray:
+        """
+        Return a 1D numpy array with value 1 for samples covered by at least one supervision,
```

Comment:
it took me a while to recognize that the difference between this and the other function was frames vs. samples.  Dunno if it messes up formatting to emphasizes those words with stars?